packages/panels/resources/views/components/sidebar/index.blade.php

```diff
@@ -15,7 +15,7 @@
         x-bind:class="
             $store.sidebar.isOpen
                 ? @js($openSidebarClasses . ' ' . 'lg:sticky')
-                : '-translate-x-full rtl:translate-x-full lg:sticky lg:translate-x-0 rtl:lg:-translate-x-0'
+                : 'z-30 -translate-x-full rtl:translate-x-full lg:sticky lg:translate-x-0 rtl:lg:-translate-x-0'
```

Comment:
`z-30` isn't needed any longer here, right?

Reply:
It is otherwise the tenancy menu will go under the table.

Reply:
But `z-30` is already in `@class()` below here?